<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateTimeEdit" name="dateTimeEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>16</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2012</year>
      <month>12</month>
      <day>18</day>
     </datetime>
    </property>
    <property name="maximumDateTime">
     <datetime>
      <hour>17</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2012</year>
      <month>12</month>
      <day>18</day>
     </datetime>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>16</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2012</year>
      <month>12</month>
      <day>18</day>
     </datetime>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>16</hour>
      <minute>59</minute>
      <second>0</second>
      <year>2012</year>
      <month>12</month>
      <day>18</day>
     </datetime>
    </property>
    <property name="maximumDateTime">
     <datetime>
      <hour>17</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2012</year>
      <month>12</month>
      <day>18</day>
     </datetime>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>16</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2012</year>
      <month>12</month>
      <day>18</day>
     </datetime>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>plot</string>
    </property>
   </widget>
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>191</width>
      <height>361</height>
     </rect>
    </property>
    <attribute name="headerVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QChartView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>491</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
